### PR TITLE
feat(renovate-config): automerge lockFileMaintenance changes

### PR DESCRIPTION
**Pull request by oscard0m**
# Description
- use 'squash' as a merge strategy (https://docs.renovatebot.com/configuration-options/#automergetype)
- merge the branch directly, do not open a PR to avoid noise to
  maintainers (https://docs.renovatebot.com/configuration-options/#automergetype)

# Context
Reduce noise and maintainers' time reviewing Renovate PRs

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
   ],
   "lockFileMaintenance": {
     "enabled": true,
-    "schedule": ["before 5am on monday"]
+    "schedule": ["before 5am on monday"],
+    "automerge": true,
+    "automergeType": "branch",
```
Comment (oscard0m) [inline default.json:12]: we can use `"automerge": "pr"` if you want to keep track Renovate is working correctly as a first step before switching to `"branch"`.

**Comment by oscard0m** [comment]
I don't have permissions to request reviewers in this repo:
@wolfy1339 @octokit/js

**Comment by wolfy1339** [comment]
I don't think us community maintainers have any access to this repo

**Comment by oscard0m** [comment]
> I don't think us community maintainers have any access to this repo

[I've contributed in the past](https://github.com/octokit/.github/pull/8), so I guess it's a matter of checking the permissions for non-maintainers for requesting a review.

**Review by gr2m — APPROVED** [review]
looks great to me 👍🏼

**Pull request merged**